styles/global-styles/fonts.css

```diff
@@ -0,0 +1,7 @@
+@font-face {
```

Comment:
One of the things I'm not sure I like is to have multiple file formats for styling. Since we'll be using css-in-js I think we should only have js files even for styling. If you want you can leave it as it is and I'll have a look on my own when this will be merged